export/Events.jsonToCsv.py

```diff
@@ -0,0 +1,23 @@
+#/bin/python
+
+import json
+
+import json
+from pprint import pprint
+
+csvString='Titel;Startdatum;SERIES;ID\n'
+
+with open('events.json') as data_file:
+    data = json.load(data_file)
+for u in data['results']:
+    Seriestitle = ''
+    if u.get('series'):
+        Seriestitle = u.get('series')['title']
+
+    csvString+=u['title'] + ';' + u['technical_start'] + ';' + Seriestitle + ';' + u['id'] + '\n'
+
+print csvString
+
+file = open('events.csv','w')
```

Comment:
Using this would be simpler and consistent with the earlier usage:

```python
with open('events.csv','w') as file:
    file.write(csvString.encode('utf-8'))
```